Userland/Services/WindowServer/Menu.h

```diff
@@ -59,6 +59,7 @@ class Menu final : public Core::Object {
     void add_item(NonnullOwnPtr<MenuItem>);
 
     DeprecatedString const& name() const { return m_name; }
+    void set_name(DeprecatedString name);
```

Comment:
The parameter name is obvious from context, just `void set_name(DeprecatedString)` is fine :)